pkg/config/args_authorization.go

```diff
@@ -0,0 +1,12 @@
+package config
+
+import (
+	"github.com/spf13/pflag"
+)
+
+var ProxyLocalAuthorization bool
+
+func AddProxyAuthorizationFlags(set *pflag.FlagSet) {
+	set.BoolVarP(&ProxyLocalAuthorization, "proxy-local-authorization", "", false,
```

Comment:
how about `--authorize-proxy-subpath`?

Reply:
Sure, it is good for me. Let me change it. Thank you.